### ISSUE TITLE
Bug in StationaryDist_FHorz_Case1_Iteration_e_raw

### ISSUE THREAD

**Issue by aledinola** [vfitoolkit/VFIToolkit-matlab#23]
In line 53 of StationaryDist_FHorz_Case1_Iteration_e_raw

simoption.loopovere

gives an error. It should be simoptions.loopovere

Same in line 94

**Comment by aledinola** [comment]
After fixing the typo, there is a new error:

Error using gpuArray/reshape
Sparse gpuArray matrices are not supported for this function.

Error in StationaryDist_FHorz_Case1_Iteration_e_raw (line 79)
            StationaryDist_jjee=reshape(Gammatranspose*StationaryDist_jjee,[N_a,N_z]);
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in StationaryDist_FHorz_Case1_e (line 28)
    StationaryDist=StationaryDist_FHorz_Case1_Iteration_e_raw(jequaloneDist,AgeWeightParamNames,Policy,N_d,N_a,N_z,N_e,N_j,pi_z_J,pi_e_J,Parameters,simoptions);
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in StationaryDist_FHorz_Case1 (line 203)
        StationaryDist=StationaryDist_FHorz_Case1_e(jequaloneDist,AgeWeightParamNames,Policy,n_d,n_a,n_z,N_j,pi_z_J,simoptions.pi_e_J,Parameters,simoptions);
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in StationaryDist_Case1_FHorz_PType (line 186)
        StationaryDist_ii=StationaryDist_FHorz_Case1(jequaloneDist_temp,AgeWeightParamNames_temp,Policy_temp,n_d_temp,n_a_temp,n_z_temp,N_j_temp,pi_z_temp,Parameters_temp,simoptions_temp);
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in main (line 243)
StatDist=StationaryDist_Case1_FHorz_PType(jequaloneDist,AgeWeightsParamNames,PTypeDistParamNames,Policy,n_d,n_a,n_z,N_j,N_i,pi_z_J,Params,simoptions);
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by robertdkirkby** [comment]
Do you have something that generates this error? 

I tried running LifeCycleModel25 which has permanent types with z and e shocks, but nothing happened, seemed to work fine. I double-checked and it does call StationaryDist_FHorz_Case1_Iteration_e_raw.

**Comment by aledinola** [comment]
Yes, see my file **AleModel24** on github. In line 151 I set simoptions.loopovere = 1; and this generates an error in the distribution. If either simoptions.loopovere = 0 or if it is not set, then the code runs without errors.

**Comment by robertdkirkby** [comment]
Thanks. I'm off on holidays, will fix once I'm back in Jan.

**Comment by robertdkirkby** [comment]
Fixed. Just needed to gather() pi_e_J.
https://github.com/vfitoolkit/VFIToolkit-matlab/commit/55291a60b4d7b47e5942f8931ee0e12fc003d85e
